src/common/webrequest.cpp

```diff
@@ -702,10 +709,10 @@ wxString wxWebResponseImpl::GetSuggestedFileName() const
     wxString contentDisp = GetHeader("Content-Disposition");
     wxWebRequestHeaderMap params;
     const wxString disp = wxPrivate::SplitParameters(contentDisp, params);
-    if ( disp == "attachment" )
+    if ( disp == "attachment" && !params["filename"].empty())
```

Comment:
Exceedingly nitpicky, but:
```suggestion
    if ( disp == "attachment" && !params["filename"].empty() )
```